web/src/components/SearchBox.vue

```diff
@@ -131,14 +131,14 @@ const suggestionMenuPosX = ref(0);
 const suggestionMenuPosY = ref(0);
 const queryTimeLimit = computed({
   get(): string | undefined {
-    const ltime = analyze(searchBox.value).ltime;
+    const ltime = analyze(searchBox.value).ltime?.[0] ?? [];
```

Comment:
I refactored the analyze function so that you can return multiple results for dictionary entries, so it supports for example multiple tags in the search field.

In case the specific tag is undefined/null (in this case) ltime is not present in the search query, return a default value (in this case an empty array)